src/Microsoft.DotNet.Interactive/Connection/ConnectStdIO.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.CommandLine;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Events;
+
+namespace Microsoft.DotNet.Interactive.Connection
+{
+    public class ConnectStdIO : ConnectKernelCommand<StdIOConnectionOptions>
+    {
+        public ConnectStdIO() : base("stdio",
+                                     "Connects to a kernel using the stdio protocol")
+        {
+            AddOption(new Option<string[]>("--command", "The command to execute")
+            {
+                AllowMultipleArgumentsPerToken = true
+            });
+            AddOption(new Option<DirectoryInfo>("--working-directory", () => new DirectoryInfo(Directory.GetCurrentDirectory()), "The working directory"));
+            AddOption(new Option<bool>("--wait-for-kernel-ready-event", () => false, "Whether to wait for a kernel ready event before continuting"));
+        }
+
+        public override Task<Kernel> CreateKernelAsync(StdIOConnectionOptions options, KernelInvocationContext context)
+        {
+            if (options.Command is null)
+            {
+                throw new ArgumentNullException("Command cannot be null");
```

Comment:
You can avoid this by setting the `--command` option's `IsRequired` property to `true`.